cli/VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Are we starting from v1.0.0 release? I guess it should be something like v0.1 at the moment?